knime_extension/src/nodes/conversion.py

```diff
@@ -0,0 +1,755 @@
+from re import S
+from typing import Callable
+import pandas as pd
+import geopandas as gp
+import knime_extension as knext
+import util.knime_utils as knut
+import pyproj as pyp
+import geopy
+from geopy.extra.rate_limiter import RateLimiter
+
+
+
+__category = knext.category(
+    path="/community/geo",
+    level_id="conversion",
+    name="Spatial Conversion",
+    description="Nodes that perform conversions from/to geometric objects.",
+    # starting at the root folder of the extension_module parameter in the knime.yml file
+    icon="icons/icon/ConversionCategory.png",
+    after="transform",
+)
+
+# Root path for all node icons in this file
+__NODE_ICON_PATH = "icons/icon/GeometryConversion/"
```

Comment:
FYI: to construct paths, there's a nice python lib `pathlib` in the standard library where Paths can be constructed as `("icons" / "icon" / "GeometryConversion").resolve()` but I guess that's overkill here as you only need to add a single filename to the end for each node.